src/internal/test/fakes/fakes.go

```diff
@@ -0,0 +1,43 @@
+package fakes
+
+import (
+	"sync"
+
+	"github.com/futurehomeno/cliffhanger/storage"
+)
+
+type fakeConfigStorage struct {
```

Comment:
This is old man grumbling: i don't like usage of fake. It's not a fake, it's a mock for testing...

Reply:
You really don't have to rename it :P